src/game/client/swarm/c_rd_boss_bar.cpp

```diff
@@ -34,6 +68,8 @@ void C_RD_Boss_Bar::PostDataUpdate( DataUpdateType_t updateType )
 	{
 		m_flBarValuePrev = m_flBarValue;
 		m_flBarValueLastChanged = gpGlobals->curtime - flSustain - flInterpolate;
+
+		SetNextClientThink( gpGlobals->curtime + 0.1f );
```

Comment:
use `CLIENT_THINK_ALWAYS` and remove the other instance of SetNextClientThink

Reply:
didnt want it to be too expensive

Reply:
Your ClientThink is cheap enough that it can run every frame without anyone noticing.

Re-scheduling might cost more than the rest of the ClientThink.